README.md

```diff
@@ -2,31 +2,102 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Create and use packages
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+We have briefly mentioned packages in passing before in previous lessons,
+like the lesson on access modifiers. Let's talk more about packages now
+and how to use them in Java.
 
-## Topic 1
+## Packages in Java
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+Every time we have created a class so far, we have not specified a "package" for
+it. In Java, **packages** are used to organize classes into "namespaces". Think
+of a namespace as a bucket filled with seashells. Every seashell collected in
+the bucket is different from all the other shells inside. The same can be said
+about classes inside a package. Every class must have a name that is unique
+within the package.  However, it is possible for
+classes in different packages to have the same name.
 
-## Topic 2
+Let's say company A, The Acme Corp, wants to implement a student grading
```

Comment:
Not very important: Consider changing 'company A' to 'a company'. I don't think it's referred to as 'Company A' in later sentences.

Reply:
fixed